tuf-spec.md

```diff
@@ -490,9 +490,11 @@ A delegated role file is located at:
 # Document formats # {#document-formats}
 
 All of the formats described below include the ability to add more
-attribute-value fields for backwards-compatible format changes.  If
-a backwards incompatible format change is needed, a new filename can
-be used.
+attribute-value fields to dictionary objects for backwards-compatible format
```

Comment:
```suggestion
key-value fields to dictionary objects for backwards-compatible format
```

Reply:
The rest of the spec is fairly consistent in referring to them as attributes and attribute-value fields.

Reply:
I think for this PR we should revert to attribute-value fields, to be consistent.

We might want to edit the entire spec to change the object related terminology. Filed as https://github.com/theupdateframework/specification/issues/175